modules/portmap.nix

```diff
@@ -0,0 +1,10 @@
+{}: {
+    domain = "fklub.dk";
+    #domain = "localhost"; # Comment the line above an uncomment this to run locally
+
+    ports = {
+        stregsystemet = "8080";
+        fikien = "8081";
```

Comment:
As far as I can tell this is completely unused, which makes sense, as DokuWiki just uses our existing Nginx instance.
```suggestion
```